.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 100
+max-line-length = 200
```

Comment:
Intentional? If so, this should probably be a separate PR because it's pretty opinionated.

Reply:
Looks like this is still present. Did we get approval from the Parsons community on doubling the maximum line length to 200?

I noticed you had an issue with a long string. Python allows you to separate strings inside parenthesis with a line break to avoid long strings breaking line length limitations. Like this:

```python
foo(f"pretend this is {a} a really long string {b}", 1, 2)

# Can be written:
foo(
  f"pretend this is {a}"
  " a really long"
  f" string {b}",
  1,
  2
)
```